harvest/api/dummy.py

```diff
@@ -18,10 +19,18 @@ class DummyStreamer(API):
     """
 
     interval_list = ['1MIN', '5MIN', '15MIN', '30MIN', '1HR', '1DAY']
+    default_now = dt.datetime(year=2021, month=7, day=19, hour=13, minute=30)
 
-    def __init__(self, path: str=None):
+    def __init__(self, path: str=None, now: dt.datetime=default_now):
```

Comment:
Might be wrong but I think this syntax is now allowed

Reply:
This syntax is fine, just double checked.